Source/santad/TTYWriter.h

```diff
@@ -15,6 +15,7 @@
 #ifndef SANTA__SANTAD__TTYWRITER_H
 #define SANTA__SANTAD__TTYWRITER_H
 
+#include <EndpointSecurity/EndpointSecurity.h>
```

Comment:
```suggestion
#import <EndpointSecurity/EndpointSecurity.h>
```

Reply:
Leaving this as `#include` as it's a C/C++ header.